Models/User.js

```diff
@@ -0,0 +1,35 @@
+const { Schema, model } = require('mongoose');
+///////////////////////////////
+// MODELS
+////////////////////////////////
+const VideoidSchema = new Schema(
+  {
+  URL: String,
+  log: String,
```

Comment:
"logs:" has to be an array of relations to the log model. To do that we need it to look like this

`
log: [
    {
        type: mongoose.Types.ObjectId, 
        ref: 'Log'
    }
]

`